Source/WTF/wtf/FastMalloc.h

```diff
@@ -656,15 +656,26 @@ void operator delete(T* object, std::destroying_delete_t, size_t size) { \
 } \
 using WTFDidOverrideDeleteForCheckedPtr = int;
 
+#define WTF_CHECKEDPTR_NO_NONNULL
+
 // Note: WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR must be declared in the most derived subclass.
 #define WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR(ClassName) \
 public: \
-    WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR_IMPL(ClassName) \
+    WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR_IMPL(ClassName, WTF_CHECKEDPTR_NO_NONNULL) \
```

Comment:
What is the case where we do not know whether the pointer is nullable or not? (Or vice versa?)

You could argue that we never know, since C++ reserves the right to `delete nullptr`. Or you could argue that we always know, since SaferCPP guidelines ask us to use RefPtr, unique_ptr, or UniqueRef (though we don't have enforcement for that right now). But I don't see yet why you would want both, or how I would choose between them.

Reply:
OK. Let me experiment with disabling that warning (in my experimental branch, initially) and if it works, the reason for this macro change may disappear entirely. I'll keep this PR open and lurking until I know the answer.